WordPress/src/main/res/values/strings.xml

```diff
@@ -935,6 +935,8 @@
     <string name="error_refresh_unauthorized_comments">You don\'t have permission to view or edit comments</string>
     <string name="error_refresh_unauthorized_pages">You don\'t have permission to view or edit pages</string>
     <string name="error_refresh_unauthorized_posts">You don\'t have permission to view or edit posts</string>
+    <string name="error_refresh_unauthorized_profile">You don\'t have permission to view or edit your profile - check your configured site is whitelisted</string>
```

Comment:
Can the user "check your configured site is whitelisted" from the app? If not, perhaps that part of the error message should be dropped (it's a bit long anyway).